src/modules/teams/components/Team.scss

```diff
@@ -0,0 +1,59 @@
+@import 'src/styles/settings/general';
+
+.image {
+    width: 8rem;    
+}
+
+.imageContainer {
+    margin-left: auto;
+    margin-right: auto;
+}
+
+h1 {
+    font-size: 0.9rem;
+    font-weight: 350;
+    color: $brand-grey-dark;
+}
+
+.teamContainer {
+    margin-left: auto;
+    margin-right: auto;
+    padding: 0.7rem;
+    text-align: center;
+    width: 90%;
+    margin-top: 1rem;
+    background-color: $brand-white;
+    box-shadow: 0px 0px 1px 0px $brand-grey;
+}
+
+.inviteContainer {
+    width: 100%;
+    float: bottom;
```

Comment:
Never heard of `float: bottom` :P? What does this do? 
I would btw not encourage the use of floats since flexbox.